sites/next.skeleton.dev/src/content/docs/design/themes.mdx

```diff
@@ -82,21 +82,116 @@ Finally, set the active theme to display using the `data-theme` attribute on you
 
 ---
 
-## Customizing Presets
+## Customize and Extend
 
-You may update the CSS Custom Properties provided by Skeleton's preset themes. Use the following syntax and apply these styles changes to your project's global stylesheet. This will target the root scope, a specific theme, then set the property value.
+### Theme Properties
+
+If you wish to update the CSS Custom Properties provided by any of Skeleton's preset themes. Apply the following changes in your global stylesheet. This will target the root scope, a specific theme, then modify each [theme property](/docs/get-started/plugin) value.
 
 {/* prettier-ignore */}
-```ts title="app.pcss"
+```css title="app.css"
 :root [data-theme='cerberus'] {
     --base-font-family: system-ui;
     --heading-font-family: system-ui;
 }
 ```
 
-## Custom Fonts
+### Target Themes
+
+If your application supports multiple themes, you can target changes to specific themes via the `data-theme` attribute.
+
+```css title="app.css"
+/** Cerberus theme h1 tags only. */
+[data-theme='cerberus'] h1 {
+    background: pink;
+}
+/** Mona theme h1 tags only. */
+[data-theme='mona'] h1 {
+    background: green;
+}
+```
+
+### Backgrounds
+
+Your application background color is set automatically using two specific theme properties. These can be overwritten in your global stylesheet using the following [theme properties](/docs/get-started/plugin#colors).
```

Comment:
Your app's background colors are set automatically using two specific theme properties. These can be overwritten using your global stylesheet.